sycl/source/detail/event_impl.hpp

```diff
@@ -329,6 +329,14 @@ class event_impl {
 
   bool isProfilingTagEvent() const noexcept { return MProfilingTagEvent; }
 
+  // Check if this event is an interoperability event.
+  bool isInterop() const noexcept {
+    // As an indication of interoperability event, we use the absence of the
+    // queue and command, as well as the fact that it can not be in enqueued
```

Comment:
"it can not be in enqueued state" or "it is not in enqueued state"?

Reply:
Fixed to "it is not in enqueued state"